libmachine/provision/boot2docker.go

```diff
@@ -182,6 +184,35 @@ func (provisioner *Boot2DockerProvisioner) GetOsReleaseInfo() (*OsRelease, error
 }
 
 func (provisioner *Boot2DockerProvisioner) Provision(swarmOptions swarm.SwarmOptions, authOptions auth.AuthOptions, engineOptions engine.EngineOptions) error {
+	const (
+		dockerPort = 2376
+	)
+
+	defer func() {
```

Comment:
Not sure if a `defer` is appropriate here, or if it's simply something we should do at the end of the function on an otherwise happy path.